src/aws-cpp-sdk-core/source/client/AWSErrorMarshaller.cpp

```diff
@@ -149,6 +194,43 @@ AWSError<CoreErrors> XmlErrorMarshaller::Marshall(const Aws::Http::HttpResponse&
     return error;
 }
 
+AWSError<CoreErrors> XmlErrorMarshaller::BuildAWSError(const std::shared_ptr<Http::HttpResponse>& httpResponse) const
```

Comment:
just checking because i know @sbera87 changed how errors were built for s3 200, is this could up to date with what he updated there?